projects/packages/my-jetpack/_inc/components/upsell-banner/index.jsx

```diff
@@ -0,0 +1,82 @@
+import { Gridicon, Button } from '@automattic/jetpack-components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import { CardWrapper } from '../card';
+
+import './style.scss';
+
+/**
+ * Upsell banner component.
+ *
+ * @param {object} props - Component props.
+ * @returns {object} UpsellBanner React component.
+ */
+const UpsellBanner = props => {
+	// todo: This component could be extracted into js-packages/components. And also the Jetpack Card component
+	const {
+		icon,
+		title,
+		description,
+		cta1Label,
+		cta1URL,
+		cta1Target,
+		cta1WithIcon,
+		cta1OnClick,
+		cta2Label,
+		cta2URL,
+		cta2Target,
+		cta2WithIcon,
+		cta2OnClick,
+	} = props;
+
+	return (
+		<CardWrapper className="upsell-banner">
+			{ icon && (
+				<div className="upsell-banner--icon">
+					<img
+						src={ icon }
+						alt={ __( 'A control panel representing Jetpack Manage', 'jetpack-my-jetpack' ) }
+					/>
+				</div>
+			) }
+			<div className="upsell-banner--content">
+				<div className="upsell-banner--content-info">
+					<h3>{ title }</h3>
+					<p>
+						{ createInterpolateElement( description, {
+							br: <br />,
+						} ) }
+					</p>
+				</div>
+				<div className="upsell-banner--content-cta">
+					{ cta1Label && cta1URL && (
+						<Button
+							className="upsell-banner--content-cta-button primary"
+							href={ cta1URL }
+							onClick={ cta1OnClick ?? null }
+							target={ cta1Target ? cta1Target : '_blank' }
+						>
+							{ cta1Label }
+							{ cta1WithIcon && <Gridicon icon="external" size={ 18 } /> }
```

Comment:
The `Button` component includes an `isExternalLink` prop that would save you some extra work here, and you wouldn't need that extra gridicon. I think it would make sense to use this here.

That may also make the props more readable, with `cta1WithIcon` becoming something like `isExternalLink`. It would become clearer how that prop is meant to be used imo.

Reply:
Also, `Button` has `externalTarget` depending on the `isExternalLink` value, so we can remove that part too.

Reply:
Updated here: https://github.com/Automattic/jetpack/pull/35078/commits/59a47c8b9f40ddec931cb9c05de7bd5a4245eb10